src/supporting-data/taxonomy/components/entry/ChildNavigation.tsx

```diff
@@ -0,0 +1,73 @@
+import { Link } from 'react-router-dom';
+import { Button } from 'franklin-sites';
+
+import { pluralise } from '../../../../shared/utils/utils';
+import {
+  getEntryPath,
+  LocationToPath,
+  Location,
+} from '../../../../app/config/urls';
+
+import { TaxonomyAPIModel } from '../../adapters/taxonomyConverter';
+import { Namespace } from '../../../../shared/types/namespaces';
+
+import styles from './styles/child-navigation.module.css';
+
+// Match the "ExpandableList" default count
+export const CHILDREN_TO_DISPLAY = 5;
+
+const ChildNavigation = ({
+  taxonId,
+  childTaxons,
+  total,
+}: {
+  taxonId: number;
+  childTaxons: TaxonomyAPIModel[];
+  total: number;
+}) => {
+  const remainingChildren = total - CHILDREN_TO_DISPLAY;
+  return (
+    <ul className="no-bullet">
+      {childTaxons.map((child) => (
+        <li key={child.taxonId}>
+          <Link to={getEntryPath(Namespace.taxonomy, child.taxonId)}>
+            {child.commonName || child.scientificName || child.taxonId}
+          </Link>
+        </li>
+      ))}
+      <li>
+        <Button
```

Comment:
Could this just be a `Link`?

Reply:
It's to get the styles of the button to reproduce the style of the ExpandableList pattern. Somehow a Button (even if it's actually a link styled with the Button's classes) would match the pattern's size whereas just a plain link wouldn't.
I'll retry that now in case I find a cleaner approach today with a fresh mind

Reply:
OK, so I had a look and I think I prefer calling the franklin Button component even if it's just in order to inject the classes (`button` and `tertiary`) instead of injecting the classes myself, because the component import is explicit whereas using global classes will make any refactor in the future more complicated.
Also, in the end it does render just an `a` tag so semantically the output is fine.